hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/event/remote/RemoteIdEventTest.java

```diff
@@ -71,12 +77,19 @@ protected RemoteIdEvent assertEntity(final long id, final RemoteIdEvent event) {
         assertThat(event.getEntityId()).isSameAs(id);
 
         RemoteIdEvent underTestCreatedEvent = (RemoteIdEvent) createProtoStuffEvent(event);
-        assertThat(underTestCreatedEvent.getEntityId()).isEqualTo(id);
+        assertDeserializeEvent(id, underTestCreatedEvent);
 
         underTestCreatedEvent = (RemoteIdEvent) createJacksonEvent(event);
-        assertThat(underTestCreatedEvent.getEntityId()).isEqualTo(id);
+        assertDeserializeEvent(id, underTestCreatedEvent);
 
         return underTestCreatedEvent;
     }
 
+    private void assertDeserializeEvent(final long id, final RemoteIdEvent underTestCreatedEvent) {
+        assertThat(underTestCreatedEvent.getEntityId()).isEqualTo(id);
```

Comment:
The `id` will always be `ENTITY_ID` so remove the parameter `id` and verify against the already existing constant